app/code/Magento/ConfigurableProduct/view/adminhtml/web/js/configurable.js

```diff
@@ -55,7 +55,7 @@ define([
             // Overwrite defaults by url
             separatorIndex = window.location.href.indexOf('#');
 
-            if (separatorIndex != -1) { //eslint-disable-line eqeqeq
+            if (separatorIndex !== -1 && !_.isEmpty(window.location.hash)) { //eslint-disable-line eqeqeq
```

Comment:
Please, add `underscore` library as a dependency in `define`.

Reply:
Hi
@omiroshnichenko 
Sorry, my mistake
It should `window.location.hash !== ''`.
I think we couldn't change the base file and add 'underscore' library because it leads to backward compatibility change

Reply:
@XxXgeoXxX Defining new or removing redundant requirejs dependency is not backward-incompatible change, so you are good to use _.isEmpty and add undescore to the dependency list.

Reply:
@omiroshnichenko 
Done